### PR TITLE
APIS-6874: integration subproject work

### PR DESCRIPTION
**Pull request by mattclark-zerogravit**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -23,7 +24,7 @@ lazy val microservice = Project(appName, file("."))
     retrieveManaged := true,
     routesGenerator := InjectedRoutesGenerator,
```
Comment (johnsgp) [inline build.sbt:25]: This line might be redundant now. I've been deleting it on other repositories without issue.

File: build.sbt [hunk 2]
```diff
@@ -65,11 +50,28 @@ lazy val microservice = Project(appName, file("."))
...
 
-  Command.command("clean-and-test") { state => "clean" :: "compile" :: "run-all-tests" :: state },
+lazy val it = (project in file("it"))
+  .enablePlugins(PlayScala)
+  .dependsOn(microservice % "test->test")
+  .settings(DefaultBuildSettings.itSettings())
+  .settings(
+    name := "integration-tests",
+    Test / testOptions += Tests.Argument(TestFrameworks.ScalaTest, "-eT"),
+    headerSettings(Test) ++ automateHeaderSettings(Test)
```
Comment (johnsgp) [inline build.sbt:61]: I think you might need a bloop setting here, which has been included in other repositories.
```  inConfig(Test)(BloopDefaults.configSettings),```

Comment (johnsgp) [reply build.sbt:61]: Comment withdrawn after discussion

**Review by johnsgp — APPROVED** [review]

File: build.sbt (hunk 1 above)
Comment (mattclark-zerogravit) [reply build.sbt]: Removed

**Review by johnsgp — APPROVED** [review]

**Pull request merged**